### PR TITLE
Install and configure SEO modules

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Summary

Google Tag Manager installed by default. Tracks anonymous users only. Update hook to import settings after install.

Metatag module installed by default. Added unique field to each feature. There are no configuration files in the /config/install/ folder, as the defaults were not modified at all. So on install of module we should have proper defaults.

Simple Sitemap module installed by default. Individual settings config for each feature added to said feature. Default config settings added to config/install and included in update hook.

Metatag and Simple Sitemap modules are listed as dependencies for all relevant features. GTM is not as it will run independent of any content types/configuration. Google Tag is also listed under the "install" config for the profile, as it is not a hard dependency and in theory could be uninstalled on any future site as required.

NOTE: svg_image dependency came along for the ride with the updated "Project" feature.

## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | no
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | no
| Did you perform browser testing? | yes
| Risk level | Medium
| Relevant links | https://thinkoomph.jira.com/browse/RIG-132
https://thinkoomph.jira.com/browse/RIG-133
https://thinkoomph.jira.com/browse/RIG-145

### CODE REVIEW

**Review by pfrilling — CHANGES_REQUESTED** [review]
I found a duplicate block of code in the update hook, otherwise, it looks good.

File: ecms_base/ecms_base.profile [hunk 1]
```diff
@@ -244,3 +244,41 @@ function ecms_base_update_9026(array &$sandbox): void {
...
+  $active_storage->write('simple_sitemap.custom_links.default', $install_source->read('simple_sitemap.custom_links.default'));
+  $active_storage->write('simple_sitemap.variants.default_hreflang', $install_source->read('simple_sitemap.variants.default_hreflang'));
+
+  /** @var \Drupal\Core\Config\FileStorage $install_source */
+  $install_source = new FileStorage($path . "/config/install/");
+
+  $active_storage->write('google_tag.settings', $install_source->read('google_tag.settings'));
+  $active_storage->write('simple_sitemap.settings', $install_source->read('simple_sitemap.settings'));
+  $active_storage->write('simple_sitemap.custom_links.default', $install_source->read('simple_sitemap.custom_links.default'));
+  $active_storage->write('simple_sitemap.variants.default_hreflang', $install_source->read('simple_sitemap.variants.default_hreflang'));
```
[the remark this answers is not in the document]
Comment (ben-hamelin) [reply ecms_base/ecms_base.profile:282]: Ugh - yes, rebased that a few times and starting glazing over :)

**Comment by ben-hamelin** [comment]
@pfrilling ok removed that

**Review by pfrilling — APPROVED** [review]
Looks good.

**Review by pfrilling — APPROVED** [review]
Still looks good once the build passes.

**Pull request merged**